core/lib/src/response/content.rs

```diff
@@ -90,5 +90,6 @@ ctrs! {
     Html: HTML, "HTML", "text/html",
     Plain: Plain, "plain text", "text/plain",
     Css: CSS, "CSS", "text/css",
-    JavaScript: JavaScript, "JavaScript", "application/javascript"
+    JavaScript: JavaScript, "JavaScript", "application/javascript",
+    Ron: RON, "RON", "text/x-ron"
```

Comment:
I couldn't find any hint on what mimetype to use for Ron, so I went with this. Only problem I noticed with this: At least firefox doesn't show the data in browser, you have to download the response as text to view it. I imagine this to be annoying in the long run when debuggin/developing.

Reply:
I believe we have stuck with only registered MIME types in the past, in which case we shouldn't add this and use something else like `text/plain` instead.

Reply:
That sounds totally reasonable, only doubt I have is that it won't be easy to determine the format the responses are sent from the outside.

Reply:
Applications can always use `Content` and set their own arbitrary content types for resources if they need to. Since browsers don't have any built-in support for RON, setting a media type has limited value in comparison to other file formats.